### PR TITLE
Fix bug on block social in footer

### LINKED ISSUE #12798: ps_socialfollow Optimization icons alignment
**Is your feature request related to a problem? Please describe.**
In full screen when you add all the social icons, the last one goes underneath.
image

The same on mobile view,
image

**Describe the solution you'd like**
The icons size are big enough to be reduced, to be in a single line.

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | All icons on one line. new design for mobile
| Type?         | bug fix 
| Category?     | FO 
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #12798 
| How to test?  | On Homepage scroll down to newsletter.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12891)

### CODE REVIEW

File: themes/classic/_dev/css/components/footer.scss [hunk 1]
```diff
@@ -50,26 +50,41 @@
...
-  }
-  a {
-    display: block;
-    height: 100%;
-    white-space: nowrap;
-    text-indent: 100%;
-    overflow: hidden;
-    &:hover {
-      color: transparent;
+  ul{
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/footer.scss]: missing space before `{`

Comment (YeLnatSs) [reply themes/classic/_dev/css/components/footer.scss]: space added

**Comment by eternoendless** [comment]
@YeLnatSs please remember to always compile your assets **in a separate commit**, so that when there are merge conflicts you can simply rebase, drop the commit including the compiled assets, compile them again, and force push them.

**Review by jolelievre — DISMISSED** [review]
Seems good, thanks @YeLnatSs

**Comment by sarahdib** [comment]
Hello @YeLnatSs the problem is solved on desktop version but not on mobile version 
Capture d’écran 2019-05-15 à 08 38 26

**Comment by YeLnatSs** [comment]
Hello @sarahdib, the problem is solved. Mobile view was made following @NadiaCamard's advice.
I can work on it again if you want to.

**Comment by sarahdib** [comment]
@YeLnatSs I saw with @TristanLDD and he said that it's ok with two line :)

**Comment by YeLnatSs** [comment]
@sarahdib Okay 😉

**Comment by PierreRambaud** [comment]
Can you please rebase and target the 1.7.6.x branch instead, the issue was tagger for the 1.7.6?

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Waiting for rebase

**Comment by YeLnatSs** [comment]
@PierreRambaud Okay no problem, I'm going to rebase it

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @YeLnatSs

**Pull request merged**